autogpt/llm/llm_utils.py

```diff
@@ -251,6 +251,12 @@ def create_embedding(
         openai.Embedding: The embedding object.
     """
     cfg = Config()
+    for plugin in cfg.plugins:
+        if plugin.can_handle_text_embedding(text, **kwargs):
+            embedding = plugin.handle_text_embedding(text, **kwargs)
+            if embedding is not None:
+                return embedding
```

Comment:
Can multiple embeddings work here? "No, that's annoying is an acceptable answer"